pages/platform/ai/deploy_guide_07_troubleshooting/guide.en-gb.md

```diff
@@ -0,0 +1,170 @@
+- Open the Monitoring dashboard of your app (Grafana dashboard is provided for each app) and check if the threshold has been reached. 
+- Refer to the following load-testing tutorial which also provides a dashboard example to follow your scaling: [AI Deploy - How to load test your application with Locust](https://docs.ovh.com/gb/en/publiccloud/ai/deploy/load-test-app/).
+
+
+### My deployed app is very slow
+
+Slowness may find its roots in multiple reasons. Indeed, each deployed app is the combination of software code and resources, such as compute and network.
+
+If you are experiencing slowness, here are some actions to investigate:
+
+- Open the Monitoring dashboard for your app (Grafana dashboard is provided for each app) and check if some resources are reaching 90/100%, such as RAM, CPU, GPU or network. You can also check the overall latency.
+- If nothing is visible, it can be an issue between the client (where the query comes) and the deployed app. As an example, if you are contacting your apps from a geographically distant point, it will add latency. Try reducing the distances in your architecture.
+- Your Docker image itself may be the root cause. Try running your Docker image locally, and query your app locally. Some apps might be heavy to run or not well optimized.
+
+### My deployment has crashed
+
+Like any cloud product, AI Deploy might experience hardware or software failures over time. To mitigate the risk on your side, please deploy your app on at least two replicas, allowing us to provide high availability. At this time, all replicas are in the same region, but it will prevent them from a physical server failure.
+
+Another root cause may be your own Docker image, for example by writing an uncontrolled amount of data into your working directory.
+
+We also recommend orchestrating your workflow with third party tools such as Airflow, Prefect, Dagster or Kestra, allowing you to relaunch an app once it has crashed.
+
+If your app crashed and you are using `ovhai CLI`, you can get more information with `ovhai app logs <app_ID>` to download logs history.
+
+### My data is not synchronized back
+
+AI Deploy does not synchronize back your remote data. Please follow [official guidelines to build & use custom Docker image](https://docs.ovh.com/gb/en/publiccloud/ai/deploy/build-use-custom-image/).
+
+## Connectivity
+
+### I don't understand how I can connect to my app
+
+AI Deploy provides an HTTP endpoint for each deployed app. You can find your endpoint via the OVHcloud control panel (*Public Cloud / AI Deploy / My app / Access URL*), API or CLI.
+
+An HTTP endpoint will look like this: `https://<unique_id>.app.gra.ai.cloud.ovh.net`
+
+Your app will be directly exposed to this HTTP endpoint and linked to a port (by default, port 8080).
+
+Depending on what you deployed, you then just have an API endpoint or a web interface. You can refer to our guide for 
```

Comment:
@baaastijn, this sentence is incomplete. Please fix it so that we can duplicate the guide. Thanks

Reply:
thanks, resolved on my side